jerry-core/ecma/builtin-objects/ecma-builtin-promise.c

```diff
@@ -196,12 +183,11 @@ ecma_builtin_promise_perform_race (ecma_value_t iterator, /**< the iterator for
 done:
   *done_p = true;
 exit:
-  ecma_deref_object (resolve_func_p);
```

Comment:
Since exit is a return now, can change `goto exit` statements to return? Less goto is better.